doc/dependency.rst

```diff
@@ -0,0 +1,23 @@
+=================== =================== ====================================
```

Comment:
should this then not be git tracked and generated during doc build?

Reply:
I am fine with keeping this under version control for now. As you wish.

Reply:
Updated `doc/conf.py` to generate the file when building the docs. (One less thing to think about for a release manager)